tests/test_sessions.py

```diff
@@ -0,0 +1,68 @@
+import sys
+from time import sleep
+from flask import json
+from tests.test_case import *
+from app.pcasts.dao import sessions_dao, users_dao
+from app import constants
+
+class SessionsTestCase(TestCase):
+
+  def setUp(self):
+    super(SessionsTestCase, self).setUp()
+    Session.query.delete()
+    db_session_commit()
+
+  def _setup_session(self):
+    user = User.query.\
+      filter(User.google_id == constants.TEST_USER_GOOGLE_ID1).\
+      first()
+    sessions_dao.get_or_create_session_and_activate(user.id)
+    session = Session.query.\
+      filter(Session.user_id == user.id).\
+      first()
+    self.assertTrue(session is not None)
+    self.assertTrue(session.user_id == user.id)
+    return user.id, \
+      session.id, \
+      session.session_token, \
+      session.update_token, \
+      session.expires_at
+
+  def test_session_update(self):
+    # Setup session
+    user_id, \
+      previous_session_id, \
+      previous_session_token, \
+      previous_update_token, \
+      previous_expires_at = self._setup_session()
+
+    sleep(2) # sleep to simulate time passing
+
+    # Update session
+    self.app.post('api/v1/sessions/update/?update_token={}'.\
+      format(previous_update_token))
+
+    updated_session = Session.query.\
+      filter(Session.user_id == user_id).\
+      first()
+
+    # Check everything
+    self.assertEqual(updated_session.id, previous_session_id)
+    self.assertTrue(updated_session.session_token != previous_session_token)
```

Comment:
`unittest.Testcase` has methods for each of these operations like `assertNotEqual` `assertGreaterThan`, etc. These work just as well though

Reply:
Oh didn't know that - will change then